java/source/substrait.rst

```diff
@@ -201,6 +201,189 @@ For example, we can join the nation and customer tables from the TPC-H benchmark
     N_NAME    NUMBER_CUSTOMER
     PERU    573
 
+Execute Filter and Projection into a Dataset
+============================================
+
+Dataset module is able to execute filters and projection with Substrait’s
+`Extended Expression`_. The substrait-java library is required in order to define
+our SQL Expression that we will pass to the dataset module.
+
+Filter Dataset
+--------------
+
+Here is an example of a Java program that filter columns from a Parquet file:
+
+- Loads a Parquet file containing the “nation” table from the TPC-H benchmark.
+- Applies a filter:
+    - `N_NATIONKEY > 10, AND`
```

Comment:
```suggestion
    - ``N_NATIONKEY > 10, AND``
```

please fix the markup

Reply:
This would probably be fine all as one line instead of making a bunch of list items